docs/data-access/data-download.md

```diff
@@ -0,0 +1,89 @@
+---
+id: data-download
+title: How to Download Data
+---
+
+The ARGO Data Platform uses the score-cient as a file download manager. Score facilitates the transfer of data with resumable download and built in BAM/CRAM slicing to make data download transfer fast and smooth.
+
+Please note:
+
+- Downloads are done in part and can be paused/resumed as needed
+- The score-client will automatically resume downloads if interrupted or paused briefly
+
+## Searching for Files
+
+Use the Platform [File Repository](https://platform.icgc-argo.org/repository) to search for a file set of interest. Query results can be narrowed down by selecting specific file values in the facet filters on the left side of the page. Once you have a file set identified, click `Download > File Manifest` on the top right of the table to download a TSV file manifest.
+
+The file manifest will contain a listing of the files that matched your query, along with some additional metatdata to assist in file identification. This file manifest will be used by the score-client as a list of files to download.
+
+> NOTE: Clinical data can be downloaded by any user. In order to download controlled molecular data, you **must have ICGC DACO approval**. Learn more about the [DACO application process here](data-access), or [apply for DACO approval here](https://icgc.org/daco).
+
+## Installing the Score-Client
+
+The score-client can be run in different ways depending on your operating system or setup:
+
+- If you are on Windows, use the Docker distribution
+- If you are on a Unix system (IOS/Linux) you can use the Docker distribution, or score-client directly.
+
+## Installing the Score-Client to Download Files
+
+### Docker Distribution
+
+Pull the latest version of the Score Docker distribution:
+
+```
+docker pull overture/score
+```
+
+Use the docker run command with the correct variables specified. You will need to define:
+
+- your personal [API Key](user-profile-and-token)
+- the file metadata Song server URL
+- the object storage Score server URL
+- the absoloute directory path where your file manifest is located
```

Comment:
typo `absoloute` (should be `absolute`)

Reply:
Would it be helpful to include the environmental variables in the list above so users understand the below command? For example:

- **ACCESSTOKEN**: your personal [API Key](user-profile-and-token) 
- **METADATA_URL**: the file metadata Song server URL 
- **STORAGE_URL**: the object storage Score server URL
...
..

Reply:
👍 great suggestion!